manifests/v/VSCodium/VSCodium/Insiders/1.85.0.23308/VSCodium.VSCodium.Insiders.installer.yaml

```diff
@@ -0,0 +1,39 @@
+# Created with WinGet Releaser v2 using Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: VSCodium.VSCodium.Insiders
+PackageVersion: 1.85.0.23308
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2023-11-04
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/VSCodium/vscodium-insiders/releases/download/1.85.0.23308-insider/VSCodiumUserSetup-arm64-1.85.0.23308-insider.exe
+  InstallerSha256: 49374932F34704FB14FE0126BA9B152C991E00272EB84986A37950EB34775653
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/VSCodium/vscodium-insiders/releases/download/1.85.0.23308-insider/VSCodiumSetup-x64-1.85.0.23308-insider.exe
+  InstallerSha256: 7ED2C7473E2784FC3C4C2880265A66EE303936D8C535ADC103399888F7053B73

```

Comment:
```suggestion
```
Hi @daiyam,

This PR has bad `x86` installers. Your previous 3 PRs for this package had the same error. Please correct your automation to not include these errant installers.